docs/using/common.md

```diff
@@ -10,7 +10,8 @@ This page describes the options supported by the startup script and how to bypas
 
 You can pass [Jupyter server options](https://jupyter-server.readthedocs.io/en/latest/operators/public-server.html) to the `start-notebook.sh` script when launching the container.
 
-1. For example, to secure the Notebook server with a custom password hashed using `IPython.lib.passwd()` instead of the default token,
+1. For example, to secure the Notebook server with a [custom password](https://jupyter-server.readthedocs.io/en/latest/operators/public-server.html#preparing-a-hashed-password)
+   hashed using `notebook.auth.security.passwd()` instead of the default token,
```

Comment:
Ah, is it even so that we should now use `jupyter_server` instead here? `notebook` is being phased out in favor of `jupyter_server` and have lots of the same capabilities.

Reply:
You are right! I was almost sure I got that function from Jupyter-server's docs.

Reply:
Ah maybe a fix for that projects documentation as well then? :) Thank you for your work in the jupyter ecosystem @irvcaza!!!

Reply:
Atm you retain the notebook reference, should it be retained or switched to jupytet_server?

Reply:
Let's change it to `jupyter_server `and do the proper changes to `jupyter_server` documentation.

Thank you for your support @consideRatio !

Reply:
from jupyter_server.auth import passwd

Should it be with .security or not here?

Reply:
Great observation! both work (I tested them)

`jupyter_server.auth.security` works because the function is defined in [that file](https://github.com/jupyter-server/jupyter_server/blob/main/jupyter_server/auth/security.py#L21) 

`jupyter_server.auth` works because (_If I understand corectly_) it is imported in the [`__init__.py`](https://github.com/jupyter-server/jupyter_server/blob/main/jupyter_server/auth/__init__.py#L4)

I prefer the second one just because it matches jupyter_server's [documentation](https://jupyter-server.readthedocs.io/en/latest/operators/public-server.html#preparing-a-hashed-password)

But if there is a stronger reason to change it to `jupyter_server.auth.security`, I have no problem with that.

Reply:
I think it's a good idea to match the docs 👍